libmscore/slur.cpp

```diff
@@ -43,7 +43,9 @@ void SlurSegment::draw(QPainter* painter) const
             case 1:
                   painter->setBrush(Qt::NoBrush);
                   pen.setWidthF(score()->styleP(Sid::SlurDottedWidth) * mag);
-                  pen.setStyle(Qt::DotLine);
+                  pen.setStyle(Qt::CustomDashLine);
+                  pen.setDashPattern(QVector<qreal> {0.01, 2.5});
```

Comment:
Shouldn't that 'magic' `2.5` rather be expressed in pen width units, `pen.widthF()` (which in turn relate to `Sid::SlurDottedWidth` and `mag`)

Reply:
![image](https://user-images.githubusercontent.com/1786669/87911643-44563300-ca6c-11ea-90fe-3fffbc5b3ed2.png)

Reply:
But then again, making the dots thicker also increases their distance, and by quite a lot. In either case...

Reply:
That other 'magic number' `0.01`can easily be just `0` BTW, and seems to mean '1 pixel'

Reply:
I think the dashed and the wide dashed slur should use a `Qt::FlatCap` BTW, to not make themselves longer by their width.

Reply:
I will take a look at that pen.widthF() for comparison. About the space between dots, I went from 2.0 to 3.0 and ended up with 2.5 but It's a matter of deciding the design of this dotted "preset". 
User can still set a custom dash pattern that looks like dotted and choose a gap - except, no rounded dots.
Ideally, Custom Dash would include an option to choose Cap Style, but that would require adding a property to be saved, and maybe set in Style... so I guess that would be a different project.

Agree with flatCaps so dots in dash-dotted lines really look like dots. Trying this next.

Reply:
I think we should be using FlatCap on all lines and RoundCap on dots, but never SquareCap as it makes the lines longer than expexted

Reply:
1.5 indeed looks doo dense, but 2.0 seems OK to me